sycl/include/sycl/ext/oneapi/sub_group_mask.hpp

```diff
@@ -255,7 +255,7 @@ struct sub_group_mask {
   sub_group_mask(const sub_group_mask &rhs)
       : Bits(rhs.Bits), bits_num(rhs.bits_num) {}
 
-  sub_group_mask &operator=(const sub_group_mask &rhs) = delete;
+  sub_group_mask &operator=(const sub_group_mask &rhs) = default;
```

Comment:
Should the copy constructor and the copy assignment operator at least be consistently either be explicitly implemented or defaulted?

Reply:
Good point. I would prefer defaulted as otherwise adding new fields would have to be explicitly added to these every time, which is error prone. If someone adds a field that needs special handling for these, they should be ready to adjust the implementation accordingly.